services/powchain/src/main/java/tech/pegasys/teku/services/powchain/PowchainConfiguration.java

```
@@ -180,6 +182,14 @@ public Builder depositSnapshotPath(final String depositSnapshotPath) {
       return this;
     }
 
+    public Builder depositSnapshotPathFromCheckpointSyncUrl(final String checkpointSyncUrl) {
+      if (StringUtils.isNotBlank(checkpointSyncUrl)) {
+        this.depositSnapshotPath =
+            Optional.of(UrlSanitizer.appendPath(checkpointSyncUrl, DEPOSIT_SNAPSHOT_URL_PATH));
+      }
```

In this case, we should throw an error. Otherwise, it will silently not use the checkpoint sync url.